policy/modules/admin/portage.te

```diff
@@ -201,6 +202,7 @@ init_exec(portage_t)
 libs_run_ldconfig(portage_t, portage_roles)
 
 miscfiles_read_localization(portage_t)
+miscfiles_relabel_man_cache(portage_t)
```

Comment:
Seems like another place where the package's build script is doing unnecessary relabeling.

Reply:
I just remembered [this comment](https://github.com/SELinuxProject/refpolicy/pull/584#issuecomment-1407808792) and that does appear to be the case. Perhaps it would be better to `dontaudit` these kinds of accesses going forward?

Reply:
Perhaps dontaudit all relabeling to take care of it completely.

Reply:
Right, since `portage` normally does a transition to `setfiles_t` during the installation phase anyway.